packages/account/src/Helpers/utils.ts

```diff
@@ -166,3 +174,6 @@ export const isDocumentNumberValid = (document_number: string, document_type: Fo
 };
 
 export const shouldHideHelperImage = (document_id: string) => document_id === IDV_NOT_APPLICABLE_OPTION.id;
+
+export const isServerError = (error: unknown): error is TServerError =>
```

Comment:
```suggestion
export const isServerError = <T,>(error: T): error is TServerError =>
```

Reply:
<img width="1092" alt="Screenshot 2023-08-23 at 08 43 24" src="https://github.com/binary-com/deriv-app/assets/103182683/4543b624-c137-414e-a36e-27bbd71f6459">
it is not working like this.
Actually, cashier team shared with me this utility function :)

Reply:
Alright :grin: :pray: